gotham/src/router/route/matcher/accept.rs

```diff
@@ -91,19 +91,24 @@ impl RouteMatcher for AcceptHeaderRouteMatcher {
     /// matcher is only able to indicate whether a successful match has been found.
     fn is_match(&self, state: &State) -> Result<(), RouteNonMatch> {
         // Request method is valid, ensure valid Accept header
-        match Headers::borrow_from(state).get::<Accept>() {
+        let headers = HeaderMap::borrow_from(state);
+        match headers.get(ACCEPT) {
             Some(accept) => {
-                for ra in accept.iter().map(|qi| &qi.item) {
-                    if *ra == mime::STAR_STAR || self.supported_media_types.contains(ra) {
-                        return Ok(());
-                    }
+                if accept == "*/*"
+                    || self.supported_media_types.contains(&accept
```

Comment:
I'm not sure but there may be a better way to do this; can't think of one now, just a note for later.